src/pe_reports/report_generator.py

```diff
@@ -352,9 +378,24 @@ def generate_reports(db, datestring, data_directory, output_directory):
             # results.)
             file = filenames[-1]
 
-            # Create folders in output directory
-            subprocess.call(["mkdir", f"{output_directory}/ppt"])
-            subprocess.call(["mkdir", f"{output_directory}/{_id}"])
+            # Create folders in output directory if folders dont exists. If the folders exists remove them and create new directory.
+            try:
+                if not os.path.exists(f"{output_directory}/ppt") or not os.path.exists(
+                    f"{output_directory}/_id"
+                ):
+                    os.mkdir(f"{output_directory}/ppt")
+                    os.mkdir(f"{output_directory}/_id")
+                else:
+                    os.remove(f"{output_directory}/ppt")
+                    os.remove(f"{output_directory}/_id")
+                    os.mkdir(f"{output_directory}/ppt")
+                    os.mkdir(f"{output_directory}/_id")
```

Comment:
This should uncomplicate the logic here.

```suggestion
            try:
                for subdir in ("_id", "ppt"):
                    if os.path.exists(f"{output_directory}/{subdir}"):
                        os.remove(f"{output_directory}/{subdir}")
                    os.mkdir(f"{output_directory}/{subdir}")
```